shoot-client/Dockerfile

```diff
@@ -17,6 +17,7 @@ FROM golang:1.19.3 AS gobuilder
 
 WORKDIR /build
 COPY ./VERSION ./VERSION
+COPY ./.git ./.git
```

Comment:
Why do we need the `.git` folder?

Reply:
Right, this is not directly related to changing the network ranges. It is fixing an error during build and was missing before. It is needed to calculate the effective version in the file `go/Makefile`: 

```makefile
EFFECTIVE_VERSION     := $(VERSION)-$(shell git rev-parse HEAD)
```